contrib/scalapblib/test/src/mill/contrib/scalapblib/TutorialTests.scala

```diff
@@ -182,6 +188,11 @@ object TutorialTests extends TestSuite {
       }
     }
 
+    "calledWithScala3Sources" - workspaceTest(TutorialWithScala3Soures) { eval =>
+      val result = eval.apply(Tutorial.core.compileScalaPB)
+      assert(result.isRight)
```

Comment:
@b-eyselein Is there something unique to the `scala3_sources` option in the generated file(s)? you could assert some specific file content.

Reply:
It mainly uses `?` instead of `_`for unknown types which generates a bunch of compiler warnings...